lib/optimizely/helpers/event_tag_utils.rb

```diff
@@ -32,7 +31,7 @@ def string_numeric?(str)
         false
       end
 
-      def get_revenue_value(event_tags)
+      def get_revenue_value(event_tags, logger)
         # Grab the revenue value from the event tags. "revenue" is a reserved keyword.
         # The revenue value must be an integer.
```

Comment:
This is not true, now we also accept string/float if they can be parsed to an int without change in value